packages/core/android/src/main/java/com/segment/analytics/reactnative/core/RNAnalyticsModule.kt

```diff
@@ -56,18 +59,32 @@ class RNAnalyticsModule(context: ReactApplicationContext): ReactContextBaseJavaM
         }
     }
 
+    private fun getOptions(properties: ReadableMap?): Options {
```

Comment:
From the invocation of this function, we pass in `options` as an argument, so should we rename this argument?

It might also be good to document this function, as it's not clear.